### ISSUE TITLE
Experimental Design : Training Script for GCN

### ISSUE THREAD

**Issue by BrandonN24** [BrandonN24/GCN-GAT-ComparativeAnalysis#9]
Create a training script for GCN using the following parameters:
Training: - Adam optimizer - 
LR: 0.01, weight decay: 5e-4 
200 epochs, early stopping 
Use NLL loss
Dropout: 0.5 
Hyperparameters:
 - Hidden dims: 64
 - Layers: 2, 3 

Ensure that the following metrics are recorded:
- Accuracy
- Training time
- Parameter Count
- Convergence
